.pre-commit-hooks.yaml

```diff
@@ -3,4 +3,5 @@
 - id: keep-sorted
   name: keep-sorted
   language: golang
+  language_version: 1.21.6
```

Comment:
Are we gonna regret hard coding this at some point? Right now, this makes us use a newer version of Go, but it will eventually get outdated and eventually may cause confusing errors as a result. We probably should check in on this later and remove it once possible.

Reply:
Potentially...and I am a little worried about our lack of ability to exercise this hook in this repo to ensure we don't accidentally break it (by updating the version in go.mod but forgetting to update it here)

I'm gunna poke around a bit to see if it is possible to exercise the hook this way in the repo so we can get a little more confidence that we'll catch big breakages

Reply:
Found a way to also have pre-commit bootstrap go for us in the local repo!